vagrant/ansible/roles/client.prep/tasks/centos7.yml

```diff
@@ -1,11 +1,11 @@
 - name: Install required packages
   yum:
     name:
-      - glusterfs-fuse
       - cifs-utils
       - samba-test
-      - python-pip
+      - python3-pip
```

Comment:
I think CentOS 7 still doesn't have Python 3+.

Reply:
Yes. I messed it up when I moved them around. I'll fix it. Thanks.

Reply:
I've just seen that the same file uses pip3 for the next task. Is that correct or should it be changed also ?

Reply:
Let's leave as it is for now 😐